pac4j-jwt/src/main/java/org/pac4j/jwt/profile/JwtGenerator.java

```diff
@@ -67,22 +67,34 @@ public JwtGenerator(final String signingSecret, final String encryptionSecret) {
         this.signingSecret = signingSecret;
         this.encryptionSecret = encryptionSecret;
     }
-
+    
     /**
      * Generates a JWT from a user profile.
      *
      * @param profile the given user profile
      * @return the created JWT
      */
     public String generate(final U profile) {
+    	return generate(profile, null, this.jwsAlgorithm);
```

Comment:
I see a backward compatibility issue here (compared to the version 1.8): previously, no `signingSecret` means no signing. Now, we don't sign JWT **by default** and this is a problem.

So in this method, you need to check if the `signingSecret` is blank and if so, pass `null`, otherwise, pass a `new MACSigner(this.signingSecret);`